static/index.html

```diff
@@ -238,20 +241,20 @@ <h3>Wireless-Tag WT32-ETH01</h3>
         </ul>
       </div>
 
-      <p>
-        <i>
-          For advanced users that want to customize the firmware: the device is
-          adoptable in
-          <a
-            href="https://my.home-assistant.io/redirect/supervisor_addon/?addon=5c53de3b_esphome&repository_url=https%3A%2F%2Fgithub.com%2Fesphome%2Fhome-assistant-addon"
-            >the ESPHome dashboard</a
-          >
-          and
-          <a href="https://github.com/esphome/bluetooth-proxies/"
-            >configuration is on GitHub</a
-          >.
-        </i>
-      </p>
+      <div>
```

Comment:
I don’t think the div adds to rendering here

Reply:
ba8407d8a1e6701f14d2b7ac3eaa6512570059b0